tests/.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+Hello! Thanks for taking interest in EGG. We are glad you're here to help us improve it!
+Please fill this templated as thoroughly as possible, that would make things much easier for us :) 
+
+<!--- Please provide a general summary of the issue in the Title above -->
+
+## Expected Behavior
+<!--- Tell us what should happen -->
+
+## Current Behavior
+<!--- Tell us what happens instead of the expected behavior -->
+
+## Steps to Reproduce
+<!--- Provide a link to a live example, or an unambiguous set of steps to -->
```

Comment:
This seems to assume that this is a bug report. I understood that issues would also be for feature reports. How about adding something like: if this is a bug...